src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLStopImpl.java

```diff
@@ -132,6 +133,11 @@
     return environment -> null;
   }
 
+  @Override
+  public DataFetcher<Object> geometries() {
+    return environment -> getValue(environment, StopLocation::getGeometry, station -> null);
```

Comment:
Should stations also have geometries so the same method could be used for stations as for stops in clients to draw them etc?

Reply:
I implemented it as a geometry collection which contains the point of the station and a convex hull of all child stops.